src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  const loadPeople = async () => {
+    setIsLoading(true);
+    setError(false);
+
+    try {
+      const loadedPeople = await getPeople();
+
+      setPeople(
+        loadedPeople.map((person, _index, peopleToMap) => ({
+          ...person,
+          mother: peopleToMap.find(mother => mother.name === person.motherName),
+          father: peopleToMap.find(father => father.name === person.fatherName),
```

Comment:
This could be improved by a bit with on running through peopleToMap with reduce and look for mother and father in one go.